src/Astreiko.homework9.Nbrb.by.Tests/APIClientTests.cs

```diff
@@ -0,0 +1,109 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Astreiko.Homework9.Nbrb.@by.API_Client;
+using Astreiko.Homework9.Nbrb.@by.API_Client.Models;
+using Astreiko.Homework9.Nbrb.@by.FileClient;
+using NUnit.Framework;
+
+namespace Astreiko.homework9.Nbrb.by.Tests
+{
+    public class APIClientTests
+    {
+        [SetUp]
+        public void Setup()
+        {
+        }
+
+        [Test]
+        public void GetRatesAsyncShouldGetRates()
+        {
+            //arrange
+            var apiClient = new APIClient();
```

Comment:
It is possible to mosk api calls. Please try to investigate these example tests
https://github.com/tms-net/NET06/commit/0947ea336e5c32a7b0f9a6a3c943cba5ec2ff09e